scripts/mainnet/deploy_root.py

```diff
@@ -15,27 +15,31 @@ def main():
     polygon_chain_id = 137
     polygon_child_name = "PolygonChild"
     polygon_childs = [
-        entry for entry in current_mainnet_registry
+        entry
+        for entry in current_mainnet_registry
         if entry.chain_id == polygon_chain_id and entry.name == polygon_child_name
     ]
     if len(polygon_childs) != 1:
         raise ValueError("Mainnet root deployment requires valid child deployment first")
-    
+
     deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
 
     # TODO: Workaround to provide PolygonChild address from existing registry
     polygon_child = polygon_childs.pop()
-    deployer.constructor_parameters.parameters["PolygonRoot"]["_fxChildTunnel"] = polygon_child.address
+    deployer.constructor_parameters.parameters["PolygonRoot"][
+        "_fxChildTunnel"
+    ] = polygon_child.address
 
     taco_application = deployer.deploy(project.TACoApplication)
 
     polygon_root = deployer.deploy(project.PolygonRoot)
 
+    deployer.transact(polygon_child.setFxRootTunnel, polygon_root.address)
```

Comment:
I realized that this step needs to be refined since it's a Polygon transaction. I'll work on a fix for that now.